Framework/Crystal/src/FindSXPeaks.cpp

```diff
@@ -128,26 +128,25 @@ void FindSXPeaks::init() {
   // Declare
   std::vector<std::string> resolutionStrategy = {
       relativeResolutionStrategy, absoluteResolutionPeaksStrategy};
-  declareProperty(
-      "ResolutionStrategy", relativeResolutionStrategy,
-      boost::make_shared<StringListValidator>(resolutionStrategy),
-      "Different options for the resolution."
-      "1. RelativeResolution: This defines a relative tolerance "
-      "needed to avoid peak duplication in number of pixels. "
-      "This selection will enable the Resolution property and "
-      "disable the XUnitResolution, PhiResolution, ThetaResolution.\n"
-      "1. AbsoluteResolution: This defines an absolute tolerance "
-      "needed to avoid peak duplication in number of pixels. "
-      "This selection will disable the Resolution property and "
-      "enable the XUnitResolution, PhiResolution, "
-      "ThetaResolution.\n");
+  declareProperty("ResolutionStrategy", relativeResolutionStrategy,
+                  boost::make_shared<StringListValidator>(resolutionStrategy),
+                  "Different options for the resolution."
+                  "1. RelativeResolution: This defines a relative tolerance "
+                  "needed to avoid peak duplication in number of pixels. "
+                  "This selection will enable the Resolution property and "
+                  "disable the XResolution, PhiResolution, ThetaResolution.\n"
+                  "1. AbsoluteResolution: This defines an absolute tolerance "
+                  "needed to avoid peak duplication in number of pixels. "
+                  "This selection will disable the Resolution property and "
+                  "enable the XResolution, PhiResolution, "
+                  "ThetaResolution.\n");
 
   declareProperty(
       "Resolution", 0.01, mustBePositiveDouble,
       "Tolerance needed to avoid peak duplication in number of pixels");
 
   declareProperty(
-      "XUnitResolution", 5000., mustBePositiveDouble,
+      "XResolution", 5000., mustBePositiveDouble,
```

Comment:
I've also just thought now. Is 5000 a good default now?

Reply:
We'll going the other way and using angstroms is also incorrect. But perhaps it is better because the resolution will be tighter by default.